### ISSUE TITLE
Soft ban with the new 1.3 update

### ISSUE THREAD

**Issue by cire27rn** [pokesniperandroid/PokeSniper-Android#34]
I am really always getting softbanned with this new update. The version 1.2 worked smooth for me. But when i used the 1.3, soft ban occured to me always. I used to enter my fake location (not jumping continents) to the home coords way back 1.2 but now i was getting soft ban. Im goin back to 1.2. Im using google acct.

**Comment by kingkai808** [comment]
I just tried catching a pokemon for a friends Google account and it kept getting pokemon fled. 1.2 worked perfect for Google. Now its all messed up. Something changed. Wonder what it is. I won't attempt anymore Google accounts now until fixed. @pokesniperandroid

**Comment by iamasri** [comment]
same with me,  i'm using ptc account.. 
i am really always getting pokemon fled with this new update v1.3

**Comment by kingkai808** [comment]
Something changed where GPS was detected during spoofs need to update asap won't use until fixed.

**Comment by kingkai808** [comment]
1.2 working like a charm for Google acct. DO NOT USE GOOGLE WITH 1.3!!!!!!

**Comment by zidonlime** [comment]
I had no problem woth 1.3,

**Comment by kingkai808** [comment]
It workes fine all day. For some reason it just started tonight for me. Never had it before.

**Comment by iamasri** [comment]
ok, the example is when i use this v1.3 i had no problem catch pokemon, but when i play normal i got softban.. even though home coord  i set same with  play normal location..

*sorry my bad english

**Comment by namecwarrior** [comment]
I think 1.3 has some bug getting soft ban when trying to snipe some pokemon. 1.2 is working fine for me. May devs can fix this. But for now i'm using 1.2.

**Comment by kingkai808** [comment]
Tested on several botted accounts that never had real GPS settings yet. Worked fine for the whole night no soft ban. Definitely a bug with returning back to current GPS. Tested on 3 botted accounts I've signed into my location. Set my location to home settings and each account got soft banned after 3 snipes. Opened up official pogo app and I couldn't catch pokemon or loot pokestops. Does not work for accounts that have GPS location already. But it did however work after 4 hours of not logging in.
